### PR TITLE
Chore: Extract out dragHandle styles

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- extracts out 3 separate variations of drag handle styling into 1
- think we can then use this styling in the scenes component as well
- might also try and extract logic to make it into a full component, but this is a good initial step i think
  - we're doing a lot of similar things in the drawer component as we're doing in the scene app splitter, would be good to standardise that logic as well
  - for example, scenes handles keyboard support nicely 🥳 

|  | before | after |
| --- | --- | --- |
| drawer | image | image |
| panel edit | image | image |
| explore | image | image |

**Why do we need this feature?**

- consistency!

**Who is this feature for?**

- everyone

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx
Comment (joshhunt) [inline public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx:78]: i think this can/should just be `const dragStyles = useStyles2(getDragStyles)`

Comment (ashharrison90) [reply public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx:78]: it's a class component

could convert it to functional in a follow up PR if you want, it doesn't look too tricky

Comment (joshhunt) [reply public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx:78]: ahhhh. true dat

**Review by Clarity-89 — APPROVED** [review]

**Review by joshhunt — APPROVED** [review]

**Comment by ashharrison90** [comment]
@grafana/explore-squad i'll wait for a review from you as this has the biggest impact on the explore drawers 👍

**Review by gelicia — APPROVED** [review]
✅ Code looks great 
✅ Makes handles more consistent (Video: left is new, right is whats on play.grafana.org)
✅ Negative lines of code add/remove ratio

10/10 PR no notes

https://github.com/grafana/grafana/assets/1533128/528e066f-250b-4b6a-be5a-29c647b04988

**Pull request merged**
